custom/ilsgateway/tanzania/handlers/register.py

```diff
@@ -28,18 +28,11 @@ def _get_facility_location(self, domain, msd_code):
         return sp['location']
 
     def _get_district_location(self, domain, sp):
-        locs = Location.view('locations/by_name',
-                             startkey=[domain, "DISTRICT", sp],
-                             endkey=[domain, "DISTRICT", sp + "z"],
-                             reduce=False,
-                             include_docs=True)
-        if len(locs) > 1:
-            locs = Location.view('locations/by_name',
-                                 startkey=[domain, "DISTRICT", sp],
-                                 endkey=[domain, "DISTRICT", sp],
-                                 reduce=False,
-                                 include_docs=True)
-        return locs[0]
+        return SQLLocation.objects.filter(
+            domain=domain,
+            location_type__name="DISTRICT",
+            name=sp,
+        )[0].couch_location
```

Comment:
Sounds good, thanks for the update.  If at some point this code stops being used, could you remove the unused code?  Every line of code has maintenance costs associated with it, such as this substitution I've done here, so we should liberally remove unneeded code.